ext/nokogiri/extconf.rb

```diff
@@ -563,18 +580,19 @@ def install
   "exslt" => ['exsltFuncRegister',      'libexslt/exslt.h'],
 }.each { |lib, (func, header)|
   have_func(func, header) ||
-  have_library(lib, func, header) ||
+  have_library(lib, func, header, iconv_libs) ||
```

Comment:
@knu - sure, but this line is the explicit issue, without a `-L/opt/local/lib` passed here compilation of `conftest.c` fails